editoast/src/converters/utils.rs

```diff
@@ -368,28 +368,31 @@ pub fn edge_to_buffer(node: &NodeId, edge: &Edge, count: i64) -> BufferStop {
     }
 }
 
+/// TODO: handle multiple overlapping electrifications
+/// Specific infrastructures can support multiple electrifications (e.g. "voltage"="600;1500;3000;15000;25000").
+/// Short term solution : pick the first one, i.g. "600;1500;3000;15000;25000" -> "600V"
```

Comment:
This should be comments and not documentation. We can move this into the function and replace the `///` by `//`.